14.js

```diff
@@ -0,0 +1,48 @@
+const findUniqueCharacters = string => {
+  const allCharacters = []
+  const repeatedCharacters = []
+
+  const array = stringToArray(string)
+
+  array.forEach(character => {
+    if (allCharacters.indexOf(character) === -1) {
+      allCharacters.push(character)
+    } else if (repeatedCharacters.indexOf(character) === -1) {
+      repeatedCharacters.push(character)
+    }
+  })
+
+  const uniqueCharacters = array.filter(character => repeatedCharacters.indexOf(character) === -1)
+
+  return arrayToString(uniqueCharacters)
+}
+
+const stringToArray = string => string.split('')
+
+const arrayToString = array => array.join('')
+
+const permutations = inputString => {
+  const result = []
```

Comment:
For clarity, I would probably refactor this to manage the exceptional (base) case first, and return immediately.  This allows us to remove some nesting that is not really necessary.  This also exposed the `temp collection, iterate, return temp collection` pattern that I typically translate into a more functional approach:
```js
const permutations = input => {
  if( input.length === 1 ) {
    return [ input ]
  }

  const letters = input.split( '' )

  return Object.keys( letters.reduce( (memo, character, index) => {
    const subString = [ ...letters.slice( 0, index ), ...this.letters.slice( index + 1 ) ].join( '' )
    
    permutations( subString ).forEach( permutation => 
      memo[ `${character}${permutation}` ] = true 
    )

    return memo
  }, {} ))
}
```
Note some of the variable renames - it's considered bad practice to put the type in the identifier. I've also omitted some of the helper functions you created, though an argument could be made that re-introducing them might clean up my implementation a little...